packages/user/src/views/VerifyEmail.tsx

```diff
@@ -93,6 +93,17 @@ const VerifyEmail = ({
       });
   };
 
+  const handleRedirect = () => {
+    const urlParameters = new URLSearchParams(window.location.search);
+    if (urlParameters) {
+      navigate(
+        `/signin?token=${window.encodeURI(
```

Comment:
should we rename `?token` to something else that gives more meaning to the param?
```suggestion
        `/signin?redirect=${window.encodeURI(
```